src/transforms.jl

```diff
@@ -1,3 +1,20 @@
+function transform_remove_local(ex)
+  ex isa Expr && ex.head === :local && return Expr(:block)
+  return ex
+end
+
+function transform_macro(ex)
+  ex isa Expr || return ex
+  ex.head !== :macrocall && return ex
+  return Expr(:call, :__secret__, ex.args)
+end
+
+function transform_macro_undo(ex)
+  ex isa Expr || return ex
+  (ex.head !== :call || ex.args[1] !== :__secret__) && return ex
+  return Expr(:macrocall, ex.args[2]...)
+end
```

Comment:
Could you please add brief docstrings explaining why these steps are necessary?

Reply:
this is gone, since it was not used anymore